src/test/resources/data/typicalDataBundle.json

```diff
@@ -768,11 +768,11 @@
       "courseId": "idOfTypicalCourse1",
       "creatorEmail": "instructor1@course1.tmt",
       "instructions": "Please please fill in the following questions.",
-      "createdTime": "2012-03-20T23:59:00Z",
-      "startTime": "2012-04-01T22:00:00Z",
-      "endTime": "2027-04-30T22:00:00Z",
-      "sessionVisibleFromTime": "2012-03-28T22:00:00Z",
-      "resultsVisibleFromTime": "2027-05-01T22:00:00Z",
+      "createdTime": "2012-04-01T23:58:00Z",
+      "startTime": "2012-04-01T23:59:00Z",
+      "endTime": "2027-04-30T21:59:00Z",
+      "sessionVisibleFromTime": "2012-04-01T23:59:00Z",
+      "resultsVisibleFromTime": "2027-05-01T21:59:00Z",
```

Comment:
- Are all these changes needed?
- `XX:59` is no longer the practice for midnight times

Reply:
Adjusted the `createdTime`, `startTime`, and `sessionVisibleFromTime` to be after the `createdTime` of the course.

Reply:
Also, use of `23:59` is just `23:58` + 1min.